esy.json

```diff
@@ -1,23 +1,28 @@
 {
   "name": "graphql_ppx",
   "description": "Used for native Reason/OCaml using latest version of OCaml",
+  "scripts": {
+    "format": "dune build @fmt --auto-promote --root .",
+    "release-static": "dune build --root . --only-packages '#{self.name}' --ignore-promoted-rules --no-config --profile release-static"
+  },
   "dependencies": {
+    "@opam/fmt": "*",
```

Comment:
I don't think this is used by the project, could be wrong. It looks like it may have snuck in before `@opam/ocamlformat` down below.